src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_LENGTH = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private final Random random = new Random();
```

Comment:
why final? In this case, nothing critical, but in the future it is better not to do this

Reply:
[https://pastenow.ru/9c735bd87760bd86b84ad5213af9c526](url)

Reply:
https://pastenow.ru/LIF83